### PR TITLE
Add add merchant center dynamic remarketing campaign example

### PR DESCRIPTION
**Pull request by AndrewMBurke**

### CODE REVIEW

File: examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+using static Google.Ads.GoogleAds.V3.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.DisplayAdFormatSettingEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AdGroupStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AssetTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// Creates a shopping campaign associated with an existing merchant center account, along
+    /// with a related ad group and dynamic display ad, and targets a user list for remarketing
```
Comment (sundquist) [inline examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs:34]: I noticed the code does responsive display ad, and this says dynamic display ad. Can we stick with one terminology, so folks don't get confused? The terminology is so similar for so many ad types.

File: examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+using static Google.Ads.GoogleAds.V3.Enums.AdvertisingChannelTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.DisplayAdFormatSettingEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AdGroupStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AssetTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// Creates a shopping campaign associated with an existing merchant center account, along
```
[the remark this answers is not in the document]
Comment (AndrewMBurke) [reply examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs:33]: Appears to be that way in some docs; updated.

File: examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs (hunk 1 above)
Comment (AndrewMBurke) [reply examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs:34]: Good point, updated the comments to replace "dynamic" with "responsive."

File: examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+            {
+                Create = campaign
+            };
+
+            // Adds the campaign.
+            MutateCampaignsResponse response = campaignServiceClient.MutateCampaigns(customerId
+                .ToString(), new[] {operation});
+            string campaignResourceName = response.Results.First().ResourceName;
+            Console.WriteLine($"Create campaign with resource name '{campaignResourceName}'.");
+            return campaignResourceName;
```
Comment (devchas) [inline examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs:170]: Nit: Create -> Created

Comment (AndrewMBurke) [reply examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs:170]: Fixed.

**Pull request merged**